lib/internal/Magento/Framework/Serialize/Serializer/Json.php

```diff
@@ -36,8 +36,8 @@ public function unserialize($string)
     {
         $result = json_decode($string, true);
         if (json_last_error() !== JSON_ERROR_NONE) {
-            throw new \InvalidArgumentException('Unable to unserialize value.');
+            throw new \InvalidArgumentException("Unable to unserialize value. Error: " . json_last_error_msg());
         }
         return $result;
-    }
+    }    
```

Comment:
Removing trailing spaces
```suggestion
    }
```